index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio skeleton</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
+      <div class="first-container">
+        <div class="font">
+          <header class="header">
+            <a class="my-logo">My Logo</a>
+            <div class="hamburger">
+              <ul class="ul-style">
+                <li>
+                  <a href="#">
+                    <img src="./images/Union.svg" alt="union background" />
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </header>
+
+          <section class="section1">
+            <h2 class="first-para">
+              I’m Mirela<br />
+              Glad to see <br />
+              you!
+            </h2>
+            <h2 class="second-para">
+              I’m a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don’t hestiate to contact
+              me.
+            </h2>
+            <p class="third-para">LET’S CONNECT</p>
+            
+          </section>
+          <div class="social_links">
+          <img src="./images/Vector-2.jpg" class="logo-icons" alt="union" />
+          <img src="./images/Linkedin-icon.jpg" class="logo-icons" alt="union" />
+          <img src="./images/Vector-3.jpg" class="logo-icons" alt="union" />
+          <img src="./images/Vector-4.jpg" class="logo-icons" alt="union" />
+          <img src="./images/Vector.jpg" class="logo-icons" alt="union" />
+          </div>
```

Comment:
- [ ] Kindly list the icons using an unordered list `ul` as requested in the project:
> The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)